docs/c-runtime-library/reference/ungetch-ungetwch-ungetch-nolock-ungetwch-nolock.md

```diff
@@ -11,7 +11,7 @@ helpviewer_keywords: ["_ungetch function", "ungetwch function", "characters, pus
 ---
 # `_ungetch`, `_ungetwch`, `_ungetch_nolock`, `_ungetwch_nolock`
 
-Pushes back the last character that's read from the console without locking.
+Pushes back the last character that's read from the console.
```

Comment:
Reverted the addition of "without locking" as this article holds both normal and nolock variants of `ungetch`.